README.md

```diff
@@ -14,18 +28,95 @@ Galleon feature-pack compatible with WildFly Preview
 
 The Maven coordinates to use is: `org.wildfly:wildfly-datasources-preview-galleon-pack:<version>`
 
-Datasources
-=======
+Supported database types
+===============
+
+For each database type its support, each feature-pack provides 3 Galleon layers that build upon each other :
+
+* One just to add the driver (eg: `postgresql-driver`).
```

Comment:
Perhaps add:

This installs a JBoss Modules module for the driver and adds a driver resource to the datasources subsystem in the server configuration.